models/public/common-sign-language-0001/common-sign-language-0001.md

```diff
@@ -0,0 +1,73 @@
+# common-sign-language-0001
+
+## Use Case and High-Level Description
+
+A human gesture recognition model for the Jester dataset recognition scenario
+(gesture-level recognition). The model uses an S3D framework with MobileNet V3 backbone.
+Please refer to the [Jester\*](https://20bn.com/datasets/jester) dataset specification
+to see the list of gestures that are recognized by this model.
+
+The model accepts a stack of frames (8 frames) sampled with a constant framerate (15 FPS)
+and produces a prediction on the input clip.
+
+## Specification
+
+| Metric                                  | Value        |
+|-----------------------------------------|--------------|
+| Top-1 accuracy (continuous Jester\*)    | 93.58%       |
+| GFlops                                  | 4.2269       |
+| MParams                                 | 4.1128       |
+| Source framework                        | PyTorch\*    |
+
+## Performance
```

Comment:
we are removing empty sections from model documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please adopt your description accordingly

Reply:
Done.